doc/users/testsuite.html

```diff
@@ -247,6 +247,16 @@
 100% tests passed, 0 tests failed out of 2
 
 Total Test time (real) =   0.43 sec</pre>
+    In some cases, additional input data has to be provided in users' tests. For
+    example, we may have an input file
+      <pre>
```

Comment:
any reason not to use <code> like we do for all other filenames in this paragraph?

Reply:
yes, to be consistent with:
```
      <pre>
CMakeLists.txt

src/CMakeLists.txt
src/step.cc
src/support.cc

tests/step/CMakeLists
tests/step/my_test_1.prm
tests/step/my_test_1.output

tests/support/CMakeLists
tests/support/my_test_2.cc
tests/support/my_test_2.output</pre>
```

Reply:
just looks better, i use `<code>` below quite a lot...

Reply:
@tjhei you OK with merging this?